static/app/views/projectsDashboard/projectCard.tsx

```diff
@@ -72,7 +131,7 @@ class ProjectCard extends Component<Props> {
                   data-test-id="project-errors"
                   to={`/organizations/${organization.slug}/issues/?project=${project.id}`}
                 >
-                  {t('errors: %s', formatAbbreviatedNumber(totalErrors))}
+                  {t('%s Errors', formatAbbreviatedNumber(totalErrors))}
```

Comment:
I think @billyvg changed this recently to not have to deal with pluralization.

Reply:
Yeah, otherwise you can have `1 Errors` -- I wonder if we can split it up to be something like `{formattedNumber} {tn('Error', unformattedNumber)}` if we wanted to keep this format. Not sure if `tn()` would work like this